dot/sync/chain_sync_integeration_test.go

```diff
@@ -657,8 +659,7 @@ func TestChainSync_doSync(t *testing.T) {
 	}
 
 	cs.network = new(syncmocks.Network)
-	cs.network.(*syncmocks.Network).On("DoBlockRequest",
-		mock.AnythingOfType("peer.ID"),
+	cs.network.(*syncmocks.Network).On("DoBlockRequest", mock.AnythingOfType("peer.ID"),
```

Comment:
nit to keep it as close as it was for git file moves tracking

```suggestion
	cs.network.(*syncmocks.Network).On("DoBlockRequest",
		mock.AnythingOfType("peer.ID"),
```

Reply:
Updated.